docs/learn/dealing-with-latency.md

```diff
@@ -295,26 +295,28 @@ Advanced games will have most of their world predicted, allowing the client and
-A common solution to solve this would be to use server authoritative gameplay with prediction and correction on both clients like described in other sections on this page. The door would be corrected, it'd teleport back open and the killing animations would play as usual. That correction would remove your players from their immersion. Not great.
-Another way could be to use action anticipation on the door closing and have the door be server driven. This way you'd remove the correction and desync problems altogether. It would still add lag to each door opening and closing though if you want your action to be instant with no animation to hide lag.
-An elegant way to handle that desync could be with this controlled desyncs technique, where you could add windows in your doors and reinterpret the kill action depending on local client state. Alice could see a knife kill while the door is still open (hasn't received the "door closed" update from the server yet) and Bob could see a gun kill through the locally closed door through the window instead. In the end, clients only care about the end "Bob is dead and killed by Alice" state. You let clients reinterpret how they reach that final state according to their own different local context.
+It can be hard to handle misprediction corrections elegantly and without taking your players out of their immersion. Clients also can't always stay in sync tick by tick, frame by frame. You can adapt your gameplay and animations to allow your clients to desync for a few seconds to resync again to smooth correction handling. By using controlled desyncs, clients can resolve and reinterpret how they reach a common state independently based on their local context. You can consider this technique as a "smart context-aware interpolation" or an "elegant correction" when predicting. It reuses some of [action anticipation](#action-anticipation)'s technique where you use non-gameplay-important animations to hide discrepancies caused by lag and reinterpret how the client will reach the target authoritative state.
+
+For example, you could have an assassination mechanic that you want to smoothly handle misprediction correction. You could have your client tell the server "I killed that target," have the server validate to prevent cheating (like with server rewind), and still give advantage to the attacker.
```

Comment:
```suggestion
For example, you could have an assassination mechanic that you want to seamlessly handle misprediction correction. You could have your client tell the server "I killed that target," have the server validate to prevent cheating (like with server rewind), and still give advantage to the attacker.
```
I think "seamless" fits more in this technique. The goal is not to smooth things out like a normal interpolation, but to actually add smart handling with animations to hide it completely, making it seamless.